lxd/storage_zfs.go

```diff
@@ -238,6 +238,18 @@ func (s *storageZfs) zfsPoolCreate() error {
 			if err := zfsPoolVolumeSet(vdev, "", "mountpoint", "none"); err != nil {
 				return err
 			}
+
+			if err := zfsPoolVolumeSet(vdev, "", "setuid", "on"); err != nil {
+				return err
+			}
+
+			if err := zfsPoolVolumeSet(vdev, "", "exec", "on"); err != nil {
+				return err
+			}
+
+			if err := zfsPoolVolumeSet(vdev, "", "devices", "on"); err != nil {
```

Comment:
We should really have a method to set multiple properties at once if that is supported by zfs.

Reply:
I don't think zfs supports modifying multiple properties at once.